manim/mobject/three_dimensions.py

```diff
@@ -32,6 +32,24 @@ def __init__(self, shade_in_3d=True, **kwargs):
 
 
 class ParametricSurface(VGroup):
+    """Creates a Parametric Surface
+
+    Exanples
+    --------
+    .. manim:: ParaSurface
+        :save_last_frame:
+
+        class ParaSurface(ThreeDScene):
+            def func(self, u, v):
+                return np.array([np.cos(u) * np.cos(v), np.cos(u) * np.sin(v), u])
+
+            def construct(self):
+                axes = ThreeDAxes()
+                surface = ParametricSurface(lambda u, v: self.func(u, v), u_min=-TAU, u_max=TAU / 2, v_min=0, v_max=2 * PI)
```

Comment:
The lambda is unnecessary here:
```suggestion
                surface = ParametricSurface(self.func, u_min=-TAU, u_max=TAU / 2, v_min=0, v_max=2 * PI)
```